docs/documentation/upgrading/topics/changes/changes-25_0_0.adoc

```diff
@@ -80,6 +80,12 @@ In order to dynamically resolve it from request headers, you need to set the `ho
 
 For more details and more comprehensive scenarios, see https://www.keycloak.org/server/hostname[Configuring the hostname (v2)].
 
+== security-admin-console Client Redirect URIs
+
+In hostname v1 the admin URL was resolved dynamically from the request if the `hostname-admin` or `hostname-admin-url` options were not set. With hostname v2 the admin URL will default instead to the frontend URL. 
+If the `hostname` option is set and `hostname-strict` is true, this change will prevent redirect URIs with alternative hostnames from working as they did previously. 
+Instead you should consider using the `hostname-admin` option instead of the redirect URIs to allow a single alternative hostname. The security-admin-console should only need the default redirect URI of `/admin/master/console/*`.
```

Comment:
```suggestion
Instead you should consider using the `hostname-admin` option instead of the redirect URIs to allow a single alternative hostname. The `security-admin-console` Client should only need the default redirect URI of `/admin/master/console/*` with Root URL of `${authAdminUrl}`.
```

Reply:
Yeah, that's true the user was not modifying the Root URL. But someone else might. I'd just add a brief mention of what's supposed to be the default value for Root URL (as in my suggestion).